Userland/Libraries/LibCore/Account.cpp

```diff
@@ -125,6 +125,35 @@ ErrorOr<Account> Account::from_uid(uid_t uid, [[maybe_unused]] Read options)
     return from_passwd(*pwd, spwd);
 }
 
+ErrorOr<Vector<Account>> Account::all([[maybe_unused]] Read options)
+{
+    Vector<Account> accounts;
+
+    ScopeGuard pwent_guard([] { endpwent(); });
+    setpwent();
+    errno = 0;
+    while (auto const* pwd = getpwent()) {
```

Comment:
This is where I'd suggest using `getpwent_r()` if we had it :^)

Reply:
Thaaat's a bit of a rabbit hole that might be better for a separate PR 😀

Reply:
This PR can now use `getpwent_r()`, since #15974 has just been merged!

Reply:
Updated to use new `getpwent_r`.